.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Tests
+on:
+  push:
+    branchs:
+      - master
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
```

Comment:
According to [this](https://github.com/derekjamerson/Blackjack_Python/actions/runs/1960330819/workflow) something about line 18 is wrong. My money says these two lines need to be dedented back to have `with:` line up with `uses`.